docs/style.css

```diff
@@ -0,0 +1,35 @@
+#idt-form {
+    background-color: darkgray;
+    max-width: 50%;
```

Comment:
Consider adding media query so that 100% width is allowed after a specific break.